src/main/java/net/mcreator/plugin/PluginLoader.java

```diff
@@ -94,6 +93,11 @@ public PluginLoader() {
 		}
 
 		this.reflections = new Reflections(new ResourcesScanner(), this);
+		checkPluginUpdates();
+	}
+
+	public static void initInstance() {

```

Comment:
Move this one to original location as it is static initializer near static instace.